lighthouse-core/lib/locales/en-XA.json

```diff
@@ -1,16 +1,238 @@
 {
+  "lighthouse-core/audits/bootup-time.js | title": {
```

Comment:
we'll probably want to stop generating this file when we get our first translation dump (this locale is automatically generated for us)